dspace-api/src/main/java/sk/dtq/dspace/app/util/ACL.java

```diff
@@ -0,0 +1,152 @@
+/**
+ * The contents of this file are subject to the license and copyright
+ * detailed in the LICENSE and NOTICE files at the root of the source
+ * tree and available online at
+ *
+ * http://www.dspace.org/license/
+ */
+package sk.dtq.dspace.app.util;
+
+import java.sql.SQLException;
+import java.util.ArrayList;
+import java.util.List;
+import java.util.Set;
+import java.util.UUID;
+import java.util.stream.Collectors;
+import java.util.stream.Stream;
+
+import org.apache.log4j.Logger;
+import org.dspace.authorize.factory.AuthorizeServiceFactory;
+import org.dspace.authorize.service.AuthorizeService;
+import org.dspace.core.Context;
+import org.dspace.eperson.EPerson;
+import org.dspace.eperson.Group;
+import org.dspace.eperson.factory.EPersonServiceFactory;
+import org.dspace.eperson.service.GroupService;
+import org.springframework.stereotype.Component;
+
+/**
+ * Class that represents Access Control List
+ *
+ * @author Michal Josífko
+ * @author milanmajchrak
```

Comment:
```suggestion
 * @author Milan Majchrak ( @ dataquest.sk)
```

Reply:
changed to @author Milan Majchrak (milan.majchrak at dataquest dot sk)